CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -1649,7 +1649,7 @@ int _CFOpenFile(const char *path, int opts) {
 }
 
 CF_CROSS_PLATFORM_EXPORT void *_CFReallocf(void *ptr, size_t size) {
-#if TARGET_OS_WIN32 || TARGET_OS_LINUX
+#if TARGET_OS_WIN32 || TARGET_OS_LINUX || defined(__OpenBSD__)
```

Comment:
Would it be worth adding a `TARGET_OS_OPENBSD`?

Reply:
I don't know if it would be worth it. There shouldn't be too many points of divergence. Here, FreeBSD has reallocf and OpenBSD doesn't.